pkg/operator/ceph/cluster/utils.go

```diff
@@ -30,28 +30,84 @@ import (
 
 // populateConfigOverrideConfigMap creates the "rook-config-override" config map
 // Its content allows modifying Ceph configuration flags
-func populateConfigOverrideConfigMap(clusterdContext *clusterd.Context, namespace string, ownerInfo *k8sutil.OwnerInfo) error {
+func populateConfigOverrideConfigMap(clusterdContext *clusterd.Context, namespace string, ownerInfo *k8sutil.OwnerInfo, clusterMetadata metav1.ObjectMeta) error {
 	ctx := context.TODO()
-	placeholderConfig := map[string]string{
-		k8sutil.ConfigOverrideVal: "",
-	}
 
-	cm := &v1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      k8sutil.ConfigOverrideName,
-			Namespace: namespace,
-		},
-		Data: placeholderConfig,
+	existingCM, err := clusterdContext.Clientset.CoreV1().ConfigMaps(namespace).Get(ctx, k8sutil.ConfigOverrideName, metav1.GetOptions{})
+	if err != nil {
+		if !kerrors.IsNotFound(err) {
+			logger.Warningf("failed to get cm %q to check labels and annotations", k8sutil.ConfigOverrideName)
+			return nil
+		}
+
+		labels := map[string]string{}
+		annotations := map[string]string{}
```

Comment:
```suggestion
		var annotations, labels map[string]string{}
```

Reply:
why add the `var`? I thought it was the same either way.

Reply:
yeah, it's same it just two line thing